### ISSUE TITLE
Dynamic Trees compatibility

### ISSUE THREAD

**Issue by MagmaSlime123** [Elite-Modding-Team/NaturaLegacy#5]
## Version Information
`Minecraft:` 1.12.2.
`Forge: ` 2860.
`Mantle:` Mantle-1.12-1.3.3.55.
`Natura:` natura-1.12.2-4.3.2.69.

## Report Information
If Dynamic Trees for Natura is installed, disabling a tree's generation should disable its Dynamic version too. (Since Apple Trees are a thing now, support for Dynamic Trees for that tree would be nice.)

## Steps to Reproduce
Title.

## Crash Log
N/A.

## Forge Log
N/A.

**Comment by MagmaSlime123** [comment]
It also seems a crash occurs upon breaking a tree from Natura's addon.

https://pastebin.com/GdWwSs2D

**Comment by ACGaming** [comment]
> It also seems a crash occurs upon breaking a tree from Natura's addon.
> 
> https://pastebin.com/GdWwSs2D

You'd need to enable stick variants in the config to resolve it for now.

**Comment by MagmaSlime123** [comment]
Ah, whoops. If they're disabled then normal sticks should drop instead then. Apple Sticks don't exist though.

**Comment by ACGaming** [comment]
Yes, we probably need to adjust the add-on at some point as we want to make stick variants entirely optional.

**Comment by ACGaming** [comment]
Since we added the apple tree as an entirely new tree variant, we're most likely forking the Dynamic Trees add-on eventually.

**Comment by IcarussOne** [comment]
The addon itself has been [forked and released](https://www.curseforge.com/minecraft/mc-mods/dramatictrees-naturalegacy), closing this issue.
